proposals/awslogs.md

```diff
@@ -0,0 +1,195 @@
+# ECS CLI Logging
+
+## Overview
+
+The following Proposal lays out a design and implementation plan for creating a user experience in the ECS CLI for getting container logs from CloudWatch.
+
+### Use Cases
+
+1. User has a known error and wants to find more info on it.
+2. User is doing a deployment and wants to tail the logs and grep for errors.
+3. User wants to quickly set up their CloudWatch Logs based upon the configuration specified in their docker compose file, and have the CLI creates any necessary log groups for them.
+4. User wants to monitor their task/service, so they continually stream the logs.
+
+## Phase 1 Solution
+Top level `ecs-cli logs` command that will not use the docker compose file. This allows it to be used by a wide array of ECS customers, not just compose users. The command will allow customers to find logs for a given task.
+
+```
+ecs-cli logs --help
+--follow                         Stream logs (continuously poll for updates)
+--task-id                        [Required] View logs for a given Task ID
+--task-def                      Required with Task ID if the task has been stopped already. Format: family:revision
+--pattern                        Substring to search for within the logs.
```

Comment:
would it make sense to call this `filter-pattern` to be more consistent with the CloudWatch Logs API? http://docs.aws.amazon.com/cli/latest/reference/logs/filter-log-events.html